solr/solrj/src/java/org/apache/solr/common/util/Utils.java

```diff
@@ -748,61 +745,6 @@ public static String applyUrlScheme(final String url, final String urlScheme) {
     return (at == -1) ? (urlScheme + "://" + url) : urlScheme + url.substring(at);
   }
 
-  /**
-   * Construct a V1 base url for the Solr node, given its name (e.g., 'app-node-1:8983_solr') and a
-   * URL scheme.
-   *
-   * @param nodeName name of the Solr node
-   * @param urlScheme scheme for the base url ('http' or 'https')
-   * @return url that looks like {@code https://app-node-1:8983/solr}
-   * @throws IllegalArgumentException if the provided node name is malformed
-   */
-  public static String getBaseUrlForNodeName(final String nodeName, final String urlScheme) {
```

Comment:
These methods are called in a bunch of places; we can't simply remove them in 9x.  They can stay and call the new location and be deprecated.